pkgs/build-support/setup-hooks/auto-patchelf.sh

```diff
@@ -1,9 +1,16 @@
 declare -a autoPatchelfLibs
+declare -Ag autoPatchelfFailedDeps
 
 gatherLibraries() {
     autoPatchelfLibs+=("$1/lib")
 }
 
+# wrapper around patchelf to raise proper error messages
+# containing the tried file name and command
+runPatchelf() {
+  patchelf $@ || (echo "Command failed: patchelf $@" && exit 1)
```

Comment:
```suggestion
  patchelf "$@" || (echo "Command failed: patchelf $*" && exit 1)
```
(at least it's better according to shellcheck)